### ISSUE TITLE
[22.0.0] requirements.txt specifies 2 different versions of diffusers[torch] which prevents installation from succeeding

### ISSUE THREAD

**Issue by ashleykleynhans** [bmaltais/kohya_ss#1565]
Line 2 and line 7 of the requirements.txt contain 2 different versions of diffusers:

```
accelerate==0.23.0
diffusers[torch]==0.21.2
# albumentations==1.3.0
aiofiles==23.2.1
altair==4.2.2
dadaptation==3.1
diffusers[torch]==0.18.2
```

This causes installation to fail.

**Comment by FurkanGozukara** [comment]
Nice catch

**Comment by mykeehu** [comment]
Which is better? It changes for me on every startup.

**Comment by FurkanGozukara** [comment]
> diffusers[torch]==0.21.2

obviously diffusers[torch]==0.21.2 would be better if working properly

**Comment by FurkanGozukara** [comment]
by the way original repo has 0.21.2 

so version 18 is error

**Comment by bmaltais** [comment]
Fixed in v22.0.1

Sorry about that one, I missed the duplicate entry when I merged kohya's dev branch into mine.

**Comment by ashleykleynhans** [comment]
Confirmed the issue is resolved in v22.01.  Thank you 🙏 .
